examples/ina219_simpletest.py

```diff
@@ -7,7 +7,7 @@
 
 i2c_bus = board.I2C()
 
-ina219 = INA219(i2c_bus)
+ina219 = INA219(i2c_bus=i2c_bus, addr=0x40)
```

Comment:
In addition to Scott's point above, the I2C address defaults to `0x40` in the library already, so you do not need to include the `addr=0x40` either. 

Please return this line to the following:
```suggestion
ina219 = INA219(i2c_bus)
```